src/cover-hero.tsx

```diff
@@ -0,0 +1,186 @@
+import * as React from 'react';
+import ResponsiveLayout from './responsive-layout';
+import Box from './box';
+import ButtonGroup from './button-group';
+import Stack from './stack';
+import {Text3, Text8} from './text';
+import {vars} from './skins/skin-contract.css';
+import * as styles from './cover-hero.css';
+import classnames from 'classnames';
+import {applyCssVars} from './utils/css';
+import * as mediaStyles from './image.css';
+import GridLayout from './grid-layout';
+import {CoverHeroMedia} from './cover-hero-media';
+import {getPrefixedDataAttributes} from './utils/dom';
+
+import type {DataAttributes} from './utils/types';
+import type {ImageProps, VideoProps} from './cover-hero-media';
+import type {AspectRatio} from './image';
+import type {ExclusifyUnion} from './utils/utility-types';
+import type {ButtonLink, ButtonPrimary, ButtonSecondary} from './button';
+import type Tag from './tag';
+import type {RendersNullableElement} from './utils/renders-element';
+import type {Variant} from './theme-variant-context';
+
+type BaseProps = {
+    headline?: RendersNullableElement<typeof Tag>;
+    pretitle?: string;
+    pretitleLinesMax?: number;
+    title: string;
+    titleLinesMax?: number;
+    titleAs?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+    description?: string;
+    descriptionLinesMax?: number;
+    extra?: React.ReactNode;
+    sideExtra?: React.ReactNode;
+    button?: RendersNullableElement<typeof ButtonPrimary>;
+    secondaryButton?: RendersNullableElement<typeof ButtonSecondary>;
+    buttonLink?: RendersNullableElement<typeof ButtonLink>;
+    minHeight?: string | number;
+    aspectRatio?: AspectRatio | number | 'auto';
+    centered?: boolean;
+    noPaddingY?: boolean;
+    /** "data-" prefix is automatically added. For example, use "testid" instead of "data-testid" */
+    dataAttributes?: DataAttributes;
+    'aria-label'?: string;
+};
+
+type BackgroundProps = {
+    background?: string;
+    variant?: Variant;
+};
+
+type CoverHeroProps = BaseProps & ExclusifyUnion<ImageProps | VideoProps | BackgroundProps>;
+
+const aspectRatioToCssString = (aspectRatio?: BaseProps['aspectRatio']) => {
+    if (!aspectRatio) {
+        return 'initial';
+    }
+    if (typeof aspectRatio === 'number' || aspectRatio === 'auto') {
+        return String(aspectRatio);
+    }
+    return aspectRatio.replace(':', ' / ');
+};
+
+const CoverHero = React.forwardRef<HTMLDivElement, CoverHeroProps>(
+    (
+        {
+            headline,
+            pretitle,
+            pretitleLinesMax,
+            title,
+            titleLinesMax,
+            titleAs = 'h1',
+            description,
+            descriptionLinesMax,
+            extra,
+            sideExtra,
+            button,
+            secondaryButton,
+            buttonLink,
+            minHeight,
```

Comment:
Hero component use `height` instead of `minHeight`. Should be the same to have parity?

Reply:
I think this is an error in Hero component, we should avoid having fixed height

Reply:
we should note it in https://github.com/Telefonica/mistica-web/issues/626